docs/conf.py

```diff
@@ -196,7 +214,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = ["_static"]
```

Comment:
So - there *is* content in the `_static` directory... but it's only being referenced by the `html_logo` definition. To avoid future confusion, we should make sure `_static` is preserved on the static path.
```suggestion
html_static_path.append("_static")
```